ostorlab_insecure_flutter_app/lib/bugs/biometric_none_cryptobject.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:local_auth/local_auth.dart';
+import 'package:ostorlab_insecure_flutter_app/bug_rule.dart';
+
+abstract BiometricNoneCryptObject class BugRule {
```

Comment:
The rule is not added to the main.

Reply:
[6653ab2](https://github.com/Ostorlab/ostorlab_insecure_android_app/pull/18/commits/6653ab2c5e080ed4f067a646b2adb951f471793a)